### DISCUSSION TITLE
How large of an attachment should I be able to send/receive?

### DISCUSSION THREAD

**Discussion by jpatters** [emersion/maddy#388] [Q&A] [answered]
I am having issues with attachments. The largest attachment I've gotten to go though successfully is 598KB. What size attachment should I be able to send/receive with the default configuration? I expected it would be 32MB based on the default value of `max_message_size`.
I am using S3 for message storage but I have also tried with fs storage and had the same result.

Thanks in advance.

**Comment by foxcpp** [comment, marked as answer]
It is a bug that was introduced in 0.5.0. With default configuration you should be able to accept messages with attachments up to ~ 31 MB.
I just pushed the relevant fix. See #389 for details.

**Reply by jpatters** [reply]
I am using `buffer fs` and still experiencing this issue.

**Reply by jpatters** [reply]
I rebuilt from master and things are working great now. Thanks, once again!
